Tribler/Core/Modules/restapi/debug_endpoint.py

```diff
@@ -342,3 +344,93 @@ def render_GET(self, request):
         request.setHeader(b'content-type', 'application/json')
         request.setHeader(b'Content-Disposition', 'attachment; filename=tribler_memory_dump_%s.json' % date_str)
         return open(dump_file_path).read()
+
+
+class DebugLogEndpoint(resource.Resource):
+    """
+    This class handles the request for displaying the logs.
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+        self.session = session
+
+    def render_GET(self, request):
+        """
+        .. http:get:: /debug/log
+
+        A GET request to this endpoint returns the location of log file
+
+            **Example request**:
+
+            .. sourcecode:: none
+
+                curl -X GET http://localhost:8085/debug/log
+
+            **Example response**:
+
+            The content of the log file, for eg.
+            INFO    1506675301.76   sqlitecachedb:181   Reading database version...
+            INFO    1506675301.76   sqlitecachedb:185   Current database version is 29
+            INFO    1506675301.76   sqlitecachedb:203   Beginning the first transaction...
+            INFO    1506675301.76         upgrade:93    tribler is in the latest version, no need to upgrade
+            INFO    1506675302.08  LaunchManyCore:254   lmc: Starting Dispersy...
+
+        """
+
+        # First, flush all the logs to make sure it is written to file
+        logger = logging.getLogger()
+        for handler in logger.handlers:
+            handler.flush()
+
+        # Get the location of log file
+        log_directory = os.path.abspath(os.environ.get('APPDATA', os.path.expanduser('~')))
+        log_directory = os.path.join(log_directory, '.Tribler', 'logs')
+        log_file = '%s/tribler-info.log' % log_directory
```

Comment:
You should use `os.path.join` since this `log_file` variable will most likely not work on Windows (on Windows, we use `\` instead of `/` as file path separator).